.jshintignore

```diff
@@ -3,6 +3,7 @@ libs
 lib-jitsi-meet.js
 lib-jitsi-meet.min.js
 lib-jitsi-meet.js.map
+karma.conf.js
```

Comment:
oops.  originally there were some symbol complaints there but now looks like it's just line lengths.  i'll fix that.